jen-jin/moment
# MVP of Resources Page

Targets: [FYDP-51](https://speaku.atlassian.net/browse/FYDP-51)

Details
- Mocking search
- Loads resources from API

Testing
- npm run/yarn start
- Navigate to /resources for your localhost
- Type in the search bar, wait for 1 second, and you should see a ""Searching..." message
- Resource contains image, category, and tags
- Clicking on "LEARN MORE" opens resource in new tab
